package.json

```diff
@@ -15,12 +15,14 @@
     "@types/node": "^14.14.6",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
+    "polished": "^4.2.2",
```

Comment:
Oh! Please remove this package also, @GustavoFSProg !
This PR is going pretty well! :muscle: